src/sp/it/pl/audio/Player.java

```diff
@@ -133,10 +110,10 @@ public static void dispose() {
 
 	/** Initialize state from last session */
 	public static void loadLastState() {
-		if (!continuePlaybackOnStart) return;
+		if (!PlayerConfiguration.Companion.getContinuePlaybackOnStart()) return;
```

Comment:
jvmstatic has a problem that it will have o be removed once the remaining of the code is ported to Kotlin. Should I still use it? We can then remove all JvmStatic annotations at once I suppose.

Reply:
Yes, I'd do that

Reply:
Just went to try and I'll refrain... There is a complication that the framework requires the IsConfig annotation on the java field, not kotlin property, because the part that reads the annotations from fields is still in Java. I do not want to mess with JvmField annotation in a kotlin singleton in order to use `@field:` speficifier to use IsConfig annotation on java field instead of the property - way too many things to go wrong.

Reply:
But the point was about JvmStatic, not JvmField?!

Reply:
IsConfig only works if it is applied on the field, requiring field: annotation, which can only be applied if there is a JvmField (in a singleton) I think. JvmField doesnt work on delegated property.

Anyway, I may have gotten confused. The class in not java class, so my arguments are moot. I'll use JvmStatic

Reply:
But: I think it is wrong idea to decorate Kotlin code in order for Java code to look better. I think all JvmStatic and JvmField annotations should be removed.